### PR TITLE
[stable/prometheus-operator] Possibility to add grafana datasources

### PR DESCRIPTION
**Pull request by etharendil**
Signed-off-by: Ondrej Kuchar <ondrej.kuchar@pan-neut.eu>

#### What this PR does / why we need it:
This PR will enable addition of user-define grafana datasources

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/prometheus-operator/values.yaml [hunk 1]
```diff
@@ -357,6 +357,22 @@ grafana:
   #   configMap: certs-configmap
   #   readOnly: true
 
+  ## Configure additional grafana datasources
+  ## ref: http://docs.grafana.org/administration/provisioning/#datasources
```
Comment (vsliouniaev) [inline stable/prometheus-operator/values.yaml:361]: This content should also be added to the ci/ directory

Comment (etharendil) [reply stable/prometheus-operator/values.yaml:361]: `additionalDataSources` added to ci dir

**Pull request merged**